src/components/SecondaryPanes/Frames.css

```diff
@@ -47,13 +47,23 @@
   margin-right: 1em;
 }
 
-.frames ul li:hover,
+.frames ul li:hover {
+  background-color: rgba(236, 236, 236,0.6);
+}
+
+:root.theme-light .frames ul li:hover .location {
+  color: #49444c;
```

Comment:
If we follow that pattern, this becomes something like `--theme-selectable-color-hover`